lib/primitives/array-element.js

```diff
@@ -92,6 +92,15 @@ module.exports = function(BaseElement, registry) {
       return undefined;
     },
 
+    /*
+     * It reduces through a given array and calls get on each item.
+     */
+    getIn: function(paths) {
```

Comment:
The name `getIn` is more from the functional programming world, specifically [Clojure](https://clojuredocs.org/clojure.core/get-in). Also see [Mori.js](http://swannodette.github.io/mori/#getIn). Will think about different name.